src/pages/ProductDetail/FavoriteBtn.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+const FavoriteBtn = ({
+  children,
+  color,
+  size,
+  outline,
+  fullWidth,
+  ...rest
+}) => {
+  return (
+    <StyledButton
+      color={color}
+      size={size}
+      outline={outline}
+      fullWidth={fullWidth}
+      {...rest}
+    >
+      {children}
+    </StyledButton>
+  );
+};
+export default FavoriteBtn;
+
+FavoriteBtn.defaultProps = {
+  color: 'white',
+  size: 'medium',
+};
+
+const colorStyles = css`
+  ${({ theme, color }) => {
+    const selected = theme.palette[color];
+    return css`
+      background: ${selected};
+      ${props =>
+        props.outline &&
+        css`
+          color: ${selected};
+          background: none;
+          border: 1px solid ${selected};
+        `}
+    `;
+  }}
```

Comment:
```suggestion
  ${({ theme, color, outline }) => css`
      background: ${theme.palette[color]};
      ${outline &&
        css`
          color: ${selected};
          background: none;
          border: 1px solid ${selected};
        `}
    `;
  }
  ```
  이런 방식으로도 표현할 수 있습니다!

Reply:
감사합니다.!